.conda/meta.yaml

```diff
@@ -1,43 +1,35 @@
 package:
   name: rbc
-  version: 0.1.0.dev2
+  version: 0.8.0
 
 source:
   git_url: https://github.com/xnd-project/rbc.git
-  git_rev: v0.1.0.dev2
+  git_rev: v0.8.0
   #git_url: ../
 
 test:
   requires:
     - pytest
   commands:
-    - pytest -sv -r s --pyargs rbc
+    - pytest -sv -r s -k "not test_python_version" --pyargs rbc
   imports:
     - rbc
     - rbc.thrift
 
 requirements:
-  build:
-    - python
-    - setuptools
   host:
+    - cffi
```

Comment:
Same here. Do we need `cffi`?

Reply:
Mistake on my side, I looked at the wrong meta file (previous release) when I started this and then forgot to update this one.